lizrabuya/trigger-pipeline-action-local
# Update Integration Tests

- update integration test workflow
- merge more conflicts
- update test commit msg
- test branch / commit defaults
- update integration tests params
- add step to verify output
